experimental/packages/api-logs/src/types/EventLogger.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { LogRecord } from './LogRecord';
+
+export interface EventLogger {
+  /**
+   * Emit a log event.
+   *
+   * @param eventName the Event name.
+   * @param logRecord the LogRecord representing the Event.
+   */
+  emitLogEvent(eventName: string, logRecord: LogRecord): void;
```

Comment:
Spec uses the name 'emit event' and I think keeping log out of the name makes it less confusing.

Reply:
Are we using the log record type intentionally? Seems like it should have an event type?

Reply:
> Spec uses the name 'emit event' and I think keeping log out of the name makes it less confusing.

This is strictly in accordance with the specification. In fact, the specification recommends "logEvent". But it's just a suggestion. Maybe what you said is right. I have changed it to "emitEvent".
![image](https://user-images.githubusercontent.com/12229677/208918722-cbe8e41c-7b09-4a3d-9c99-44bb94eb59c6.png)

Reply:
> Are we using the log record type intentionally? Seems like it should have an event type?

As above, here is strictly in accordance with the specification too, these two guys go together. If  we don't highlight the log, we really need an Event Type. But the two of them are actually one data model. I added a 'type EventRecord = LogRecord' and change  api to `emitEvent(eventName: string, eventRecord: EventRecord): void;`,  here can discuss whether the `eventName` should be placed in the `EventRecord` type